src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is the filename `EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml` intended? (same issue with corresponding `.bib`)

Reply:
Thanks for spotting. Overlooked that